core/dbt/cli/main.py

```
@@ -576,6 +577,36 @@ def run(ctx, **kwargs):
     return results, success
 
 
+# dbt run
+@cli.command("retry")
+@click.pass_context
+@p.project_dir
+@p.profiles_dir
+@p.vars
+@p.profile
+@p.target
+@p.state
+@p.threads
+@p.fail_fast
```

We should just get fail fast option from the previous args in run results.json?